plugins/cliconf/eos.py

```diff
@@ -153,7 +153,7 @@ def get_session_config(
             cmd = line["command"]
             if cmd == "end":
                 continue
-            if cmd.startswith("banner") or multiline:
+            if cmd.startswith(("banner", "code unit")) or multiline:
```

Comment:
Starting with `code unit` will not capture all the RCF multiline code units. We also have the singular unnamed code unit whose EOS config cmd is simply `code` instead of `code unit FOO`.

Reply:
Since there is no context awareness here, we would then catch all lines starting with code no matter where in the CLI they are placed. Can you guarantee that we do not have code anywhere else? or maybe code is on a line by itself?

Reply:
I can't guarantee that we do not have "code" in any other CLI command. In the absense of mode/context awareness, I think checking for "code" on a line by itself is probably the best option.